### PR TITLE
Basic route command

### LINKED ISSUE #15: Add basic route command
add a route command parser and a route command class to execute the route command.

### PR DESCRIPTION
**Pull request by wamikamalik**
Add basic command without handling exceptions/errors
Fix #15

### CODE REVIEW

File: src/main/java/seedu/duke/BusStops.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
     MUSEUM("Museum", "Opposite Faculty of Engineering"),
     UNIVERSITYTOWN("University Town","place full of fun"),
     UNIVERSITYHEALTHCENTRE("University Health Centre","UHC, where students can get MC"),
-    OPPKENTRIDGEMRTSTATION("OppKentRidgeMRTstation","MRT"),
+    OPPKENTRIDGEMRTSTATION("Opp Kent Ridge MRT station","MRT"),
     RAFFLESHALL("Raffles Hall","Hall located nearby Faculty of Engineering"),
     KENTVALE("Kent Vale","somewhere"),
```
Comment (mrwsy1) [inline src/main/java/seedu/duke/BusStops.java]:
```suggestion
    KENTVALE("Kent Vale","Accommodation for visiting faculty guests"),
```

File: src/main/java/seedu/duke/ui/Ui.java [hunk 2]
```diff
@@ -30,4 +33,18 @@ public static void commandList() {
...
+        for (Bus option: options) {
+            if (option.toString() != null) {
+                System.out.println(i + ". " + option.toString());
+                i++;
+            }
+        }
+        if (i == 1) {
+            System.out.println("none.\n\tThere are no direct bus routes. :(");
+        }
+    }
```
Comment (yuxinwu99) [inline src/main/java/seedu/duke/ui/Ui.java]: Might be better to separate i to a counter and a boolean for better clarity.

File: src/main/java/seedu/duke/BusStops.java [hunk 3]
```diff
@@ -12,7 +12,7 @@
     MUSEUM("Museum", "Opposite Faculty of Engineering"),
     UNIVERSITYTOWN("University Town","place full of fun"),
```
Comment (mrwsy1) [inline src/main/java/seedu/duke/BusStops.java]:
```suggestion
    UNIVERSITYTOWN("University Town","Place full of fun"),
```

File: src/main/java/seedu/duke/BusStops.java (hunk 1 above)
Comment (wamikamalik) [reply src/main/java/seedu/duke/BusStops.java]: Hahah, okie! I will push these changes in a new commit!

File: src/main/java/seedu/duke/ui/Ui.java (hunk 2 above)
Comment (wamikamalik) [reply src/main/java/seedu/duke/ui/Ui.java]: Okie! I will follow that approach.

**Pull request merged**